services/workflows-service/src/common/utils/unified-api-client/unified-api-client.ts

```diff
@@ -116,4 +116,16 @@ export class UnifiedApiClient {
       overrideSchemas,
     });
   }
+
+  public async createCustomer(payload: Customer) {
+    return await this.axiosInstance.post('/customers', payload);
+  }
+
+  public async updateCustomer(id: string, payload: Customer) {
+    return await this.axiosInstance.put(`/customers/${id}`, payload);
+  }
+
+  public async deleteCustomer(id: string) {
+    return await this.axiosInstance.delete(`/customers/${id}`);
+  }
```

Comment:
One thing which i think is missing is a "sync" method

We sometimes change stuff in DB (till we have all editing capabilities in the admin dashboard), and if we can then invoke sync (that syncs wf service and unified API customer) via API it will solve some burdens

it will also help with the initial migration I guess (we can just invoke sync of all current customers)